kolibri/deployment/default/urls.py

```diff
@@ -26,4 +26,6 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^$', include('kolibri.core.urls')),
     url(r'', include('kolibri.content.urls')),
+    url(r'^api/', include('kolibri.auth.urls')),
```

Comment:
What is the distinction between api and api-auth? I agree that a convention is good for resources. Specifying them with the <appname> seems totally reasonable to me. The content app will need to be made compliant with this.